nucleus/constants.py

```diff
@@ -1,5 +1,4 @@
-# NUCLEUS_ENDPOINT = "https://api.scale.com/v1/nucleus"
-NUCLEUS_ENDPOINT = "http://localhost:3000/v1/nucleus"
+NUCLEUS_ENDPOINT = "https://api.scale.com/v1/nucleus"
```

Comment:
😬 Have to be careful to not edit this in master, glad this PR is fixing.

We should probably use an environment variable to let python client users override the default endpoint URL, rather than always changing this constant on local branches for testing